cmd/lightnode/lightnode.go

```diff
@@ -66,6 +66,13 @@ func main() {
 
 	options.Whitelist = conf.Whitelist
 
+	// Hacky, but needs to be done to override whats in the darknode
```

Comment:
How come?

Reply:
Previously the lightnode would support all selectors from the darknode, irrespective of the RPCs you set. This allows you to limit the amount of watchers you run. It's only hacky in the sense that most of the other flags are parsed before the whitelist is fetched from the darknode; but we need to override their presence based on this flag so it becomes a bit more convoluted to do it in a more structured manner

Reply:
```suggestion
	// Replace Darknode whitelist with a custom one if it is set.
```